### DISCUSSION TITLE
CA

### DISCUSSION THREAD

**Discussion by sengtech** [microsoftgraph/msgraph-bicep-types#69] [General]
Hi, any plans to support the creation of Conditional Access policies?

Thanks

**Comment by dkershaw10** [comment]
Thanks @sengtech - we have CA policy as one of the highest priority backlog items, for support as Bicep resources.  I don't yet have an ETA for this, but will update you when I do.

Do you have any scenarios for where you want to control and manage deployments of CA policies?

**Reply by sengtech** [reply]
Thanks, @dkershaw10 for your response. I understand and am excited about this capability. There are many scenarios where deploying and managing CA policies through DevOps tooling would enhance security and governance. This would be especially beneficial for customers with multiple tenants, such as a Conditional Access Security Baseline.

**Comment by dkershaw10** [comment]
I'm closing this discussion with a link to the tracking issue #87
